exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
-const name = " Daniel  ";
+const name = "Nagehan";
+const nameLength=name.length;
 
-console.log(message);
+console.log(`My name is ${name} and my name is ${nameLength} chracters long` );
```

Comment:
exercise 2 should be using the .trim() function
try starting with something like `const name = "       Nagehan       "`